include/openthread/backbone_router_ftd.h

```diff
@@ -193,6 +193,110 @@ void otBackboneRouterConfigNextDuaRegistrationResponse(otInstance *
+ *
+ */
+typedef enum
+{
+    OT_BACKBONE_ROUTER_MULTICAST_LISTENER_ADDED   = 0, ///< Multicast Listener was added.
+    OT_BACKBONE_ROUTER_MULTICAST_LISTENER_REMOVED = 1, ///< Multicast Listener was removed or expired.
+} otBackboneRouterMulticastListenerEvent;
+
+/**
+ * This function pointer is called whenever the Multicast Listeners change.
+ *
+ * @param[in] aContext  The user context pointer.
+ * @param[in] aEvent    The Multicast Listener event.
+ * @param[in] aAddress  The Ip6 multicast address of the Multicast Listener.
+ *
+ */
+typedef void (*otBackboneRouterMulticastListenerCallback)(void *                                 aContext,
+                                                          otBackboneRouterMulticastListenerEvent aEvent,
+                                                          const otIp6Address *                   aAddress);
+
+/**
+ * This method sets the Backbone Router Multicast Listener callback.
+ *
+ * @param[in] aInstance  A pointer to an OpenThread instance.
+ * @param[in] aCallback  A pointer to the Multicast Listener callback.
+ * @param[in] aContext   A user context pointer.
+ *
+ */
+void otBackboneRouterSetMulticastListenerCallback(otInstance *                              aInstance,
+                                                  otBackboneRouterMulticastListenerCallback aCallback,
+                                                  void *                                    aContext);
+
+/**
+ * This method clears the Multicast Listeners.
+ *
+ * Note: available only when `OPENTHREAD_CONFIG_REFERENCE_DEVICE_ENABLE` is enabled.
+ *       Only used for test and certification.
+ *
+ * @param[in] aInstance A pointer to an OpenThread instance.
+ *
+ * @sa otBackboneRouterMulticastListenerAdd
+ * @sa otBackboneRouterMulticastListenerGetNext
+ *
+ */
+void otBackboneRouterMulticastListenerClear(otInstance *aInstance);
+
+/**
+ * This method adds a Multicast Listener.
+ *
+ * Note: available only when `OPENTHREAD_CONFIG_REFERENCE_DEVICE_ENABLE` is enabled.
+ *       Only used for test and certification.
+ *
+ * @param[in] aInstance  A pointer to an OpenThread instance.
+ * @param[in] aAddress   The Multicast Listener address.
+ * @param[in] aTimeout   The timeout (in milliseconds) of the Multicast Listener, or 0 to use the default MLR timeout.
+ *
+ * @sa otBackboneRouterMulticastListenerClear
+ * @sa otBackboneRouterMulticastListenerGetNext
+ *
+ */
+otError otBackboneRouterMulticastListenerAdd(otInstance *aInstance, const otIp6Address *aAddress, uint32_t aTimeout);
+
+#define OT_BACKBONE_ROUTER_MULTICAST_LISTENER_ITERATOR_INIT \
+    0 ///< Initializer for otBackboneRouterMulticastListenerIterator
+
+typedef uint16_t otBackboneRouterMulticastListenerIterator; ///< Used to iterate through Multicast Listeners.
+
+/**
+ * This structure represents a Backbone Router Multicast Listener info.
+ *
+ */
+typedef struct otBackboneRouterMulticastListenerInfo
+{
+    otIp6Address mAddress;
+    uint32_t     mTimeout;
```

Comment:
```suggestion
    uint32_t     mTimeoutInMilliseconds;
```

I think putting the scale with the timeout value is more readable. So the reader don't need to do a research to find out the scale by themselves. Thoughts?

Reply:
By the way, I remember that MLR.req is measuring the timeout value in `Seconds`. Is there a good reason to use `Milliseconds` at here?